pkgs/test/overriding.nix

```diff
@@ -3,12 +3,42 @@
 let
   tests =
     let
-      p = pkgs.python3Packages.xpybutil.overridePythonAttrs (_: { dontWrapPythonPrograms = true; });
+      inherit (pkgsAllowingBroken.python3Packages) pillow xpybutil;
```

Comment:
Is the choice important here? If so I would use something like pip or maybe cryptography but not an orchide like pillow or xpyutil.

Reply:
> Is the choice important here? If so I would use something like pip or maybe cryptography but not an orchide like pillow or xpyutil.

`xpyutil` appears in a previous test case, and `pillow` is one of its dependencies. There's no special reason to choose them. `pip` and `cryptography` sounds like a good idea, as they are not expected to break.

Reply:
Addressed.